audio/out/ao_pipewire.c

```diff
@@ -571,6 +586,12 @@ static int init(struct ao *ao)
 
     pw_thread_loop_unlock(p->loop);
 
+    while (p->init_state == init_state_none)
+        pw_thread_loop_wait(p->loop);
+
```

Comment:
This is a tight blocking loop. Are we already running on the ao thread at this point?

Reply:
The ao playthread is only used for push AOs, which ao_pipewire is not.

Would a timelimit for the loop work? This should happen fairly fast.

Reply:
@philipl The loop now waits for 50 milliseconds at most. (And could be reduced further)

Reply:
Ok. That seems workable.